css/emailverification.css

```diff
@@ -0,0 +1,41 @@
+
+body{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background: rgba(196, 196, 196, 0.17);  
+}
+
+.display{
+  height: 350px;
+  background: #1F59BB;
+}
+
+.emailVer-container{
+  margin-top: 3rem;
+  height: 800px;
+  padding: 30px;
+  background: #FFFFFF;
+  color: #1F59BB;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#1F59BB" to be "#1f59bb" (color-hex-case)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5674265)